index.bs

```diff
@@ -736,6 +736,16 @@ Each {{XRSpace}} has a <dfn for="XRSpace">native origin</dfn> that is tracked by
 
 The [=effective origin=] of an {{XRSpace}} can only be observed in the coordinate system of another {{XRSpace}} as an {{XRPose}}, returned by an {{XRFrame}}'s {{XRFrame/getPose()}} method. The spatial relationship between {{XRSpace}}s MAY change between {{XRFrame}}s.
 
+Each {{XRSpace}} has an <dfn for="XRSpace">internal type</dfn>, used to internally distinguish the various kinds of spaces. This value can be <dfn for="XRSpace">viewer space</dfn>, <dfn for="XRSpace">input target ray</dfn>, <dfn for="XRSpace">input grip</dfn>, or one of the variants of {{XRReferenceSpaceType}}.
```

Comment:
I wasn't sure if this was always going to be the case (that there's a fixed offset between the ray space and the grip space). The language around this would become a bit indirect, though (having to say that the origin offset is such that the effective origin matches up with the different axes).

Overall I kinda like having different types for different spaces even if they can be related to each other. They become easier to talk about overall, which is kinda the point of introducing the internal type. If we go for this, should we keep the distinction between eye-level and floor-level?